psqlgraph/node.py

```diff
@@ -64,37 +65,43 @@ def edges_out(self):
 
     @classmethod
     def __declare_last__(cls):
-        src_ids, dst_ids = [], []
-        for scls in Edge.get_subclasses():
-            name = scls.__name__
-            name_in = '_{}_in'.format(name)
-            name_out = '_{}_out'.format(name)
-            src_assoc = getattr(scls, SRC_DST_ASSOC)
-            dst_assoc = getattr(scls, DST_SRC_ASSOC)
-            if scls.__dst_class__ == cls.__name__:
-                if not hasattr(cls, name_in):
-                    edge_in = relationship(
-                        name,
-                        foreign_keys=[scls.dst_id],
-                        backref='dst',
-                        cascade='all, delete, delete-orphan',
-                    )
-                    setattr(cls, name_in, edge_in)
-                    cls._edges_in.append(name_in)
-                    dst_ids.append(scls.dst_id)
-                cls._set_association_proxy(scls, dst_assoc, name_in, 'src')
-            if scls.__src_class__ == cls.__name__:
-                if not hasattr(cls, name_out):
-                    edge_out = relationship(
-                        name,
-                        foreign_keys=[scls.src_id],
-                        backref='src',
-                        cascade='all, delete, delete-orphan',
-                    )
-                    setattr(cls, name_out, edge_out)
-                    cls._edges_out.append(name_out)
-                    src_ids.append(scls.src_id)
-                cls._set_association_proxy(scls, src_assoc, name_out, 'dst')
+        """ Execute once after all mappings has been configured by sqlalchemy
+            Maps edges to nodes based on definitions
+        """
+        for ncls in Node.get_subclasses():
```

Comment:
given a similar change here, I guess the answer to my other question is "yes" 🤣

Reply:
Yea, its only called once now, not for each sub class that extends this